extensions/sql-database-projects/src/sqldbproj.d.ts

```diff
@@ -324,7 +324,7 @@ declare module 'sqldbproj' {
 		sqlEdge = 'Azure SQL Edge',
 		sqlDwServerless = 'Azure Synapse Serverless SQL Pool',
 		sqlDwUnified = 'Synapse Data Warehouse in Microsoft Fabric',
-		sqlDbFabric = 'Fabric Mirrored SQL Database (Preview)'
+		sqlDbFabric = 'SQL database in Fabric (preview)'
```

Comment:
I'm sure the lower-case 'd' is intended, though it reads like a proper noun here to me, so it looks a bit off.

Reply:
that is unfortunately correct as "SQL database in Fabric"
https://learn.microsoft.com/en-us/fabric/database/sql/overview